tests/linearizability/linearizability_test.go

```diff
@@ -43,77 +45,112 @@ const (
 func TestLinearizability(t *testing.T) {
 	testRunner.BeforeTest(t)
 	tcs := []struct {
-		name      string
-		failpoint Failpoint
-		config    e2e.EtcdProcessClusterConfig
+		name        string
+		failpoint   FailpointConfig
+		config      e2e.EtcdProcessClusterConfig
+		clientCount int
+		trafficType Traffic
 	}{
 		{
-			name:      "ClusterOfSize1",
-			failpoint: RandomFailpoint,
+			name: "ClusterOfSize1",
+			failpoint: FailpointConfig{
+				failpoint:           RandomFailpoint,
+				count:               failpointTriggersCount,
+				waitBetweenTriggers: waitBetweenFailpointTriggers,
+			},
 			config: *e2e.NewConfig(
 				e2e.WithClusterSize(1),
 				e2e.WithGoFailEnabled(true),
 				e2e.WithCompactionBatchLimit(100), // required for compactBeforeCommitBatch and compactAfterCommitBatch failpoints
 			),
 		},
 		{
-			name:      "ClusterOfSize3",
-			failpoint: RandomFailpoint,
+			name: "ClusterOfSize3",
+			failpoint: FailpointConfig{
+				failpoint:           RandomFailpoint,
+				count:               failpointTriggersCount,
+				waitBetweenTriggers: waitBetweenFailpointTriggers,
+			},
 			config: *e2e.NewConfig(
 				e2e.WithGoFailEnabled(true),
 				e2e.WithCompactionBatchLimit(100), // required for compactBeforeCommitBatch and compactAfterCommitBatch failpoints
 			),
 		},
 		{
-			name:      "Issue14370",
-			failpoint: RaftBeforeSavePanic,
+			name: "Issue14370",
+			failpoint: FailpointConfig{
+				failpoint:           RaftBeforeSavePanic,
+				count:               failpointTriggersCount,
+				waitBetweenTriggers: waitBetweenFailpointTriggers,
+			},
 			config: *e2e.NewConfig(
 				e2e.WithClusterSize(1),
 				e2e.WithGoFailEnabled(true),
 			),
 		},
+		{
+			name: "Issue14571",
+			failpoint: FailpointConfig{
+				failpoint: MemberAddFailpoint,
+				count:     1,
+			},
+			config: e2e.EtcdProcessClusterConfig{
+				ClusterSize:   1,
+				SnapshotCount: 2,
+			},
+			clientCount: 3,
```

Comment:
Why this requires 3 clients?

Reply:
Only 3 clients are needed to reproduce on v3.5.5.